addons/mail/static/src/models/chat_window.js

```diff
@@ -566,6 +576,10 @@ registerModel({
             default: false,
             compute: '_computeHasCallButtons',
         }),
+        hasCloseAsBackButton: attr({
+            default: false,
+            compute: '_computeHasCloseAsBackButton',
```

Comment:
I usually sort the field attributes alphabetically. Do we have some kind of guidelines for this? I see that our way of doing things is inconsistent, at least in this file.